src/cli/providers/messenger/attachment.js

```diff
@@ -83,7 +83,7 @@ const logUploadInfo = uploadInfo => {
 };
 
 export async function uploadAttachment(ctx) {
-  const { f, force: _force, y, yes: _yes, t, token: _token } = ctx.argv;
+  const { f, force: _force, y, yes: _yes, t, token } = ctx.argv;
```

Comment:
We should rename `_force`, `_yes` too.

Reply:
We use `force` & `yes` as variable for ctx.argv boolean result.

```js
const force = f || _force;
const yes = y || _yes;
```

That's why we need to rename `force` as `_force`, `yes` as `_yes`.
And we use `accessToken` for the `token` & `t` boolean result, so we don't need to rename `token`.

Reply:
or maybe change all codebase to the following convention:

```js
const force = ctx.argv.f || ctx.argv.force;
const yes = ctx.argv.y || ctx.argv.yes;
const token = ctx.argv.t || ctx.argv.token;

const accessToken = token;
```

@chentsulin what do you think?